spec/models/veteran_onboarding_spec.rb

```diff
@@ -20,10 +22,31 @@
     end
   end
 
+  it 'creates a VeteranOnboarding object if toggle is enabled' do
+    Flipper.enable(:veteran_onboarding_beta_flow, user)
+    expect { VeteranOnboarding.for_user(user) }.to change(VeteranOnboarding, :count).by(1)
+  end
+
   describe '#show_onboarding_flow_on_login' do
     it 'returns the value of display_onboarding_flow' do
-      subject = described_class.new(display_onboarding_flow: true, user_account:)
+      subject = described_class.for_user(user)
+      Flipper.enable(:veteran_onboarding_beta_flow, user)
       expect(subject.show_onboarding_flow_on_login).to be_truthy
     end
+
+    # default logic to show onboarding is 180 days, can be overridden by setting value in
+    # Settings.veteran_onboarding.onboarding_threshold_days
```

Comment:
https://github.com/department-of-veterans-affairs/va-iir/issues/786 - set the value in application manifests